README.md

```diff
@@ -39,6 +39,20 @@ Places to ask questions and find other community resources.
 - [EU Working Group "APIs for Innovative Public Services (API4IPS)"](https://ec.europa.eu/cefdigital/wiki/pages/viewpage.action?pageId=254313406)
   - [REST best practices](https://ec.europa.eu/cefdigital/wiki/pages/viewpage.action?pageId=335282435) by that working group
 
+## Governmental and federal ambitions to establish more OSS
+Projects, working groups and policy papers concerning OSS in public administration.
+
+### Germany
+- Deutsche Cloud Strategie / cloud strategy germany
+  https://www.cio.bund.de/Web/DE/Innovative-Vorhaben/Deutsche-Verwaltungscloud-Strategie/deutsche_verwaltungscloud_strategie_node.html
+- Zentrum für digitale Souveränität / digital sovereignity centre
+  https://www.cio.bund.de/Web/DE/Innovative-Vorhaben/ZenDiS/zendis_node.html
+- Souveräner Arbeitsplatz / sovereign workplace
+  https://www.cio.bund.de/Web/DE/Innovative-Vorhaben/Souveraener-Arbeitsplatz/souveraener_arbeitsplatz_node.html
+  - [Linux Arbeitsplatz Schleswig-Holstein (Studie)](https://www.schleswig-holstein.de/DE/landesregierung/themen/digitalisierung/linux-plus1/Downloads/_dateien/linux-studie.html)
+- Webbasierter Arbeitsplatz des Projekt Phoenix der Dataport / Phoenix project, a webbased workplace by Dataport
+  https://www.dphoenixsuite.de/
```

Comment:
This is a SaaS hosting service. We should address some questions here: Which foss software ist hosted here? How is this different from other hosting services?